software/rfcs/rocketFind.md

```diff
@@ -0,0 +1,62 @@
+# RocketFind
+
+RocketFind is our system for finding rockets once they've landed.
+
+## Pre-reading
+
+We've tried several GPS based approaches to finding rockets in the past. 
+
+In 2021, we had a custom avionics solution which would send us a text message with the GPS coordinates. Unfortunately we weren't able to launch APEX I then, and due to the temperamental nature of the avionics (mostly the SIM800L board), we decided to replace it with an off the shelf solution.
+
+In 2022, we used a Vodafone Curve tracker to find the rocket. The Curve would allow us to access the rocket's GPS coordinates via an app. The Curve service has a small monthly fee. We encountered some issues with this system when we tried to find APEX I. 
+
+1. We didn't receive any coordinates for 40ish minutes, most likely due to the patch network conditions at SARA. 
+
+2. We received a 2nd set of coordinates that would place the rocket 12km away in a nearby town. We aren't exactly sure why this happened, but the leading theory is that the Curve attempted to predict it's location, by using past readings.
+
+## Why
+
+We need to find the rocket in a Scottish moor.
+
+## Proposal
+
+This is a faily open ended proposal, since I don't know what the best solution is, nor do I know if I've covered every possible solution to the problem. What I am going to do is discuss several different ideas that we could use to locate the rocket faster.
+
+## General approach
+
+All of the below proposals have the same underlying approach - have something that broadcasts a signal, and use the signal strength to allow people to locate the device. A key advantage of this approach is that it allows us to find the rocket regardless of the network conditions at the landing site, since we're only sending a signal and not sending data over a network. The tradeoff to not connecting to a network is that we'll have to be within a certain distance of the rocket to locate it, which would depend on the type of radio broadcast device we use.
+
+Note: None of these should be integrated with the main avionics, so we can iterate on this, without changing the firmware of the main avionics.
+
+### Approach 1 - Bluetooth
```

Comment:
not sure what UKRA would have to say about this approach - I have a feeling they'd definitely want some type of GPS data

Reply:
Yeah that's a good point, maybe we should ask em if they have any recommendations since they've been launching rockets for ages